pkg/core/statesync/module.go

```diff
@@ -459,6 +537,76 @@ func (s *Module) AddMPTNodes(nodes [][]byte) error {
 	return nil
 }
 
+// AddContractStorageData adds a batch of key-value pairs for storage-based sync.
+func (s *Module) AddContractStorageData(kvs []storage.KeyValue, syncHeight uint32, expectedRoot util.Uint256) error {
```

Comment:
Let's use `AddContractStorageItems`, I checked it in the code and it looks more intuitive.